connector/src/main/scala/com/google/cloud/spark/bigquery/SparkBigQueryOptions.scala

```diff
@@ -130,6 +132,8 @@ object SparkBigQueryOptions {
       parameters, Seq("maxParallelism", "parallelism"))
       .map(_.toInt)
     val temporaryGcsBucket = getAnyOption(normalizedAllConf, parameters, "temporaryGcsBucket")
+    val persistentGcsBucket = getAnyOption(normalizedAllConf, parameters, "persistentGcsBucket")
+    val persistentGcsPath = getAnyOption(normalizedAllConf, parameters, "persistentGcsPath")
```

Comment:
Please change to `getOption(parameters, "persistentGcsPath")` - it doesn't make sense for the path to be shared among DataFrames (unlike the bucket)